rotkehlchen/accounting/structures/base.py

```diff
@@ -427,7 +433,11 @@ def process(
             )
             return 1
 
-        return 1
+        # TODO: change to a proper solution. Right now all generic history events are processed as
+        # if they were evm events. This works since default accounting rules identify events only
+        # by event type + subtype and don't pay attention to counterparty, but we should properly
```

Comment:
There is
```
This works since default accounting rules identify events only
        # by event type + subtype and don't pay attention to counterparty
```

that I think should make this clear. If to rephrase it, how would you do it?

Reply:
```suggestion
        # if they were evm events. In order to apply the correct accounting rules for historyEvents we use
        # the default accounting rules that are chosen only using the type and the subtype of the event.
```